src/app/ubs-admin/components/ubs-admin-orders/ubs-admin-orders.component.ts

```diff
@@ -0,0 +1,33 @@
+import { AdminOrdersService } from './../../services/admin-orders.service';
+import { Component, OnInit } from '@angular/core';
+import { takeUntil } from 'rxjs/operators';
+import { Subject } from 'rxjs';
+
+@Component({
+  selector: 'app-ubs-admin-orders',
+  templateUrl: './ubs-admin-orders.component.html',
+  styleUrls: ['./ubs-admin-orders.component.scss']
+})
+export class UbsAdminOrdersComponent implements OnInit {
+  destroy: Subject<boolean> = new Subject<boolean>();
+  orders = [];
+  isAnyOrders: boolean;
+
+  constructor(private adminOrdersService: AdminOrdersService) {}
+
+  ngOnInit() {
+    this.adminOrdersService
+      .getOrders()
+      .pipe(takeUntil(this.destroy))
+      .subscribe((item) => {
+        console.log(item);
+        this.orders = item;
+      });
+  }
+
+  changeCard(id) {
+    this.orders.map((el) => {
+      el.id === id && el.extend ? el.extend = false : el.id === id ? el.extend = true : el.extend = el.extend;
+    });
```

Comment:
this will return an array of undefined ? no ?

Reply:
you should return el

Reply:
No. This map return the same array, but with changed property "extend". 
By the way that property is mine. So, in the future, when we get real data, the property and method will change.